Assets/Scripts/ScreenShotScript.cs

```diff
@@ -0,0 +1,73 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEditor;
+using System.Collections;
+
+public class ScreenShotScript : MonoBehaviour {
+
+	public AudioSource audio;
+	public RawImage screenShot;
+	
+	private Texture2D screenShotTexture;
+	private string lastScreenShotPath;
+	
+	// Use this for initialization
+	void Start () {
+		screenShotTexture = new Texture2D(100, 100);
+		screenShot.texture = screenShotTexture;
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
+	
+	void TakeScreenShot (string path) {
```

Comment:
Callback methods such as Start and Update are defined in superclasses, but TakeScreenshot seems not to be defined in superclasses.

http://docs.unity3d.com/ScriptReference/MonoBehaviour.html